lib/win32/certstore/store_base.rb

```diff
@@ -84,18 +84,23 @@ def cert_list(store_handler)
       # store_handler => Open certificate store handler
       # certificate_thumbprint => thumbprint is a hash. which could be sha1 or md5.
       def cert_delete(store_handler, certificate_thumbprint)
+        validate_thumbprint(certificate_thumbprint)
+        cert_name = memory_ptr
+        thumbprint = update_thumbprint(certificate_thumbprint)
+        cert_pem = get_cert_pem(thumbprint)
+        cert_pem = format_pem(cert_pem)
```

Comment:
This is a little confusing, It might be better to do `cert_pem = format_pem(get_cert_pem(thumbprint))` instead.

Reply:
That's great thanks a lot.